### DISCUSSION TITLE
smoothness control

### DISCUSSION THREAD

**Discussion by yashrajbharti** [hiukim/mind-ar-js#99] [Q&A]
I have been working on a project using Mind AR, I wanted to know how to play with smooth count, smoothness threshold etc values. (Is it not the same way we do in AR.js)

**Comment by hiukim** [comment]
There are some kind of smoothness parameters internally, but at the moment they are not exposed and configurable. But I think it could be a good idea to do that.

**Reply by mo-adel99** [reply]
i think that would be a great feature to add, but is there any work arounds currently to make the tracking smoother?

**Comment by yashrajbharti** [comment]
Good to know, thanks. In my case, I need the least smooth count values so there's no "lazy effect" or say the AR 3d model catches up with the movement late.. I'm okay if the AR model totally disappears when the device has a lot of movements. And then come back at it's place when the device is stable. I'm currently trying a work around using AR engine start and stop.. but it will be a lot more easier with smooth threshold etc. The AR model are 3D arrows I want them to point at the right stuff or just not ti point at all. Thanks for your response. It really helps 🙌

**Reply by juanfajardodv** [reply]
Hi @yashrajbharti were you able to set the smoothness parameters?

**Reply by mo-adel99** [reply]
any luck reducing that lazy effect?
